tests/test_utils.py

```diff
@@ -470,6 +470,33 @@ def test_parse_dict_header(value, expected):
     assert parse_dict_header(value) == expected
 
 
+@pytest.mark.parametrize(
+    'value, expected', (
+        (
+            'application/xml',
+            ('application/xml', dict())
+        ),
+        (
+            'application/json ; charset=utf-8',
+            ('application/json', {'charset': 'utf-8'})
+        ),
+        (
+            'text/plain',
+            ('text/plain', dict())
+        ),
+        (
+            'multipart/form-data; boundary = something ; \'boundary2=something_else\' ; no_equals ',
```

Comment:
Can you also add a test case with double-quotes around the value? ie: `boundary = "something";`

Reply:
Still would like to see a test-case containing this scenario.